stub-idp/src/test/java/uk/gov/ida/stub/idp/configuration/BaseConfigurationTest.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.ida.stub.idp.configuration;
+
+import io.dropwizard.configuration.ConfigurationSourceProvider;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class BaseConfigurationTest {
```

Comment:
Yes, I guess this could be `ConfigurationTestHelper` or something
basically I wanted to reuse the code from `StubIdpConfigurationTest` in `DatabaseConfigurationTest`.